### ISSUE TITLE
no-default-features: can't find crate deflate

### ISSUE THREAD

**Issue by FauxFaux** [PistonDevelopers/image-png#107]
For:
```toml
[dependencies.png]
default-features = false
version = "0.14"
```

I get:

```
   Compiling png v0.14.0
error[E0463]: can't find crate for `deflate`
 --> /home/yeet/.cargo/registry/src/github.com-1ecc6299db9ec823/png-0.14.0/src/common.rs:2:1
  |
2 | extern crate deflate;
  | ^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error
```

Seems to have been added by 477679295b20a6a7d6e9713b8e8efd720643ca79, and definitely looks incorrect to me; `common.rs` says "shared", so can't have non-default stuff like that in it?

(I'm on modern Ubuntu, recent nightly Rust. Almost certainly irrelevant.)

**Comment by HeroicKatora** [comment]
Should be fixed in https://github.com/PistonDevelopers/image-png/commit/c2b57a52ebad8308c4c01860038730f7e88ec1c5. Sorry for not having gone through a PR, I accidentally pushed this to `master` instead of another branch as I thought I had pointed the repo at my fork...

**Comment by FauxFaux** [comment]
Hah! I was planning to PR it, but my cat had interfered. Thanks.

How about a `.travis.yml` entry to test this case?

**Comment by FauxFaux** [comment]
Haha, beaten again.

I'm going to bed.

**Comment by HeroicKatora** [comment]
> That release fixes #107 (accidentally pushed without PR in c2b57a5) and introduces a new travis configuration to test a build without features in the future.

;)
